### PR TITLE
Complete "community overview" page migration to handboock

### PR DESCRIPTION
**Pull request by jasonblais**
Includes a few updates from legacy page: https://docs.mattermost.com/process/community-overview.html

Related docs PR: https://github.com/mattermost/docs/pull/3582

### CODE REVIEW

File: contributors/contributors/community.md [hunk 1]
```diff
@@ -59,27 +63,27 @@ There are many ways to contribute:
...
 * **SaaS-based Integration Solutions** - Includes connectivity and integration with services like Twitter, Zendesk, Salesforce and hundreds of other extensions with services like Zapier.
-* **Programming Solutions** - Includes language-specific drivers and connectors to simplify the work of other developers
+* **Programming Solutions** - Includes language-specific drivers and connectors to simplify the work of other developers.
 * **Mattermost User Experience Plug-Ins** - Includes server and client-side plug-ins that enhance the native capabilities of a Mattermost server, such as voice, video and screensharing with Zoom’s self-hosted or vendor-hosted solutions.
-* **Federated Experiences** - Includes bridges with IRC, Slack and other messaging applications through solutions like Matterbridge
-* **Custom User Experiences** - Includes integrations into Terminal and Pidgin built off the same core APIs as Mattermost’s Web, Desktop, iOS and Android experiences
+* **Federated Experiences** - Includes bridges with IRC, Slack and other messaging applications through solutions like Matterbridge.
```
Comment (justinegeffen) [inline contributors/contributors/community.md]:
```suggestion
* **Federated Experiences** - Includes bridges with IRC, Slack, and other messaging applications through solutions like Matterbridge.
```

File: contributors/contributors/community.md [hunk 2]
```diff
@@ -59,27 +63,27 @@ There are many ways to contribute:
...
+* **Custom User Experiences** - Includes integrations into Terminal and Pidgin built off the same core APIs as Mattermost’s Web, Desktop, iOS and Android experiences.
```
Comment (justinegeffen) [inline contributors/contributors/community.md]:
```suggestion
* **Custom User Experiences** - Includes integrations into Terminal and Pidgin built off the same core APIs as Mattermost’s Web, Desktop, iOS, and Android experiences.
```

File: contributors/contributors/community.md [hunk 3]
```diff
@@ -109,7 +113,7 @@ Community Coordinator: Jason Blais
 
 Documentation Contributors update, edit and create new documentation, help text, training and other materials to help the community get the most out of Mattermost software. Core documentation is found at [https://docs.mattermost.com](https://docs.mattermost.com/) and contributions can be made via the open source repository at [https://github.com/mattermost/docs/](https://github.com/mattermost/docs/)
```
Comment (justinegeffen) [inline contributors/contributors/community.md]:
```suggestion
Documentation Contributors update, edit and create new documentation, help text, training and other materials to help the community get the most out of Mattermost software. Core documentation is found at [https://docs.mattermost.com](https://docs.mattermost.com/) and contributions can be made via the open source repository at [https://github.com/mattermost/docs/](https://github.com/mattermost/docs/).
```

File: contributors/contributors/community.md [hunk 4]
```diff
@@ -121,21 +125,21 @@ Community Coordinator: TBD
 
 #### [Knowledge Base Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
 
-Knowledge Base Contributors share feedback, questions and answers on Mattermost through forums for [trouble shooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), Github issues and the [Mattermost community server](https://community.mattermost.com/).
+Knowledge Base Contributors share feedback, questions and answers on Mattermost through forums for [trouble shooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), GitHub issues and the [Mattermost community server](https://community.mattermost.com/).
```
Comment (justinegeffen) [inline contributors/contributors/community.md]:
```suggestion
Knowledge Base Contributors share feedback, questions, and answers on Mattermost through forums for [trouble shooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), GitHub issues, and the [Mattermost community server](https://community.mattermost.com/).
```

File: contributors/contributors/community.md [hunk 5]
```diff
@@ -121,21 +125,21 @@ Community Coordinator: TBD
...
 
-Knowledge Base Contributors share feedback, questions and answers on Mattermost through forums for [trouble shooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), Github issues and the [Mattermost community server](https://community.mattermost.com/).
+Knowledge Base Contributors share feedback, questions, and answers on Mattermost through forums for [trouble shooting](https://www.mattermost.org/troubleshoot/), [feature proposals](https://www.mattermost.org/feature-ideas/), GitHub issues, and the [Mattermost community server](https://community.mattermost.com/).
 
 Community Coordinator: Amy Blais
 
 #### [Deployment Solution Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
 
 Deployment Solutions Contributors create packaging and automation for making it easier to deploy Mattermost, using tools such as Docker, Kubernetes and other leading virtualization and orchestration solutions. See [Deployment Solutions Programs](https://docs.mattermost.com/guides/orchestration.html) for guidance on how to build to common Mattermost standards.
```
Comment (justinegeffen) [inline contributors/contributors/community.md]:
```suggestion
Deployment Solutions Contributors create packaging and automation for making it easier to deploy Mattermost, using tools such as Docker, Kubernetes, and other leading virtualization and orchestration solutions. See [Deployment Solutions Programs](https://docs.mattermost.com/guides/orchestration.html) for guidance on how to build to common Mattermost standards.
```

File: contributors/contributors/community.md [hunk 6]
```diff
@@ -59,27 +63,27 @@ There are many ways to contribute:
 
 #### [Solution Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
 
-As of July 2018, [over 1000 open source projects on GitHub reference Mattermost](https://github.com/search?q=mattermost). Solution Builders create a host of technologies to empower Mattermost software to solve real world problems. Technologies include:
+As of April 2019, [over 1,800 open source projects on GitHub reference Mattermost](https://github.com/search?q=mattermost). Solution Builders create a host of technologies to empower Mattermost software to solve real world problems. Technologies include:
 
-* **Self-hosted Integration Solutions** - Includes connectivity and integration with 3rd party applications such as Jira, Confluence, GitLab, Jenkins and other tools
+* **Self-hosted Integration Solutions** - Includes connectivity and integration with 3rd party applications such as Jira, Confluence, GitLab, Jenkins and other tools.
```
Comment (justinegeffen) [inline contributors/contributors/community.md]:
```suggestion
* **Self-hosted Integration Solutions** - Includes connectivity and integration with third-party applications such as Jira, Confluence, GitLab, Jenkins, and other tools.
```

File: contributors/contributors/community.md [hunk 7]
```diff
@@ -59,27 +63,27 @@ There are many ways to contribute:
...
 * **SaaS-based Integration Solutions** - Includes connectivity and integration with services like Twitter, Zendesk, Salesforce and hundreds of other extensions with services like Zapier.
```
Comment (justinegeffen) [inline contributors/contributors/community.md]:
```suggestion
* **SaaS-based Integration Solutions** - Includes connectivity and integration with services like Twitter, Zendesk, Salesforce, and hundreds of other extensions with services like Zapier.
```

File: contributors/contributors/community.md [hunk 8]
```diff
@@ -59,27 +63,27 @@ There are many ways to contribute:
...
-* **Programming Solutions** - Includes language-specific drivers and connectors to simplify the work of other developers
+* **Programming Solutions** - Includes language-specific drivers and connectors to simplify the work of other developers.
 * **Mattermost User Experience Plug-Ins** - Includes server and client-side plug-ins that enhance the native capabilities of a Mattermost server, such as voice, video and screensharing with Zoom’s self-hosted or vendor-hosted solutions.
```
Comment (justinegeffen) [inline contributors/contributors/community.md]:
```suggestion
* **Mattermost User Experience Plug-Ins** - Includes server and client-side plug-ins that enhance the native capabilities of a Mattermost server, such as voice, video, and screensharing with Zoom’s self-hosted or vendor-hosted solutions.
```

File: contributors/contributors/community.md [hunk 9]
```diff
@@ -107,9 +111,9 @@ Community Coordinator: Jason Blais
 
 #### [Documentation Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
 
-Documentation Contributors update, edit and create new documentation, help text, training and other materials to help the community get the most out of Mattermost software. Core documentation is found at [https://docs.mattermost.com](https://docs.mattermost.com/) and contributions can be made via the open source repository at [https://github.com/mattermost/docs/](https://github.com/mattermost/docs/)
+Documentation Contributors update, edit and create new documentation, help text, training and other materials to help the community get the most out of Mattermost software. Core documentation is found at [https://docs.mattermost.com](https://docs.mattermost.com/) and contributions can be made via the open source repository at [https://github.com/mattermost/docs/](https://github.com/mattermost/docs/).
```
Comment (justinegeffen) [inline contributors/contributors/community.md]:
```suggestion
Documentation Contributors update, edit, and create new documentation, help text, training, and other materials to help the community get the most out of Mattermost software. Core documentation is found at [https://docs.mattermost.com](https://docs.mattermost.com/) and contributions can be made via the open source repository at [https://github.com/mattermost/docs/](https://github.com/mattermost/docs/).
```

**Comment by justinegeffen** [comment]
Looks good to me. I'd like to confirm with @andrewbrown00 as to whether the links in the UX section are correct, or should be updated to a more current set of documentation.

File: contributors/contributors/community.md [hunk 10]
```diff
@@ -121,21 +125,21 @@ Community Coordinator: TBD
...
+Deployment Solutions Contributors create packaging and automation for making it easier to deploy Mattermost, using tools such as Docker, Kubernetes, and other leading virtualization and orchestration solutions. See [Deployment Solutions Programs](https://docs.mattermost.com/guides/orchestration.html) for guidance on how to build to common Mattermost standards.
 
-Community Coordinator: Amy Blais
+Community Coordinator: Jason Blais and Amy Blais
 
 #### [UX Design Contributors](https://docs.mattermost.com/process/community-overview.html#contents)
 
 UX Design Contributors influence the design of the Mattermost user experience through discussion in the `ux-design` channel on the Mattermost continous integration server at [https://community.mattermost.com/core/channels/ux-design](https://community.mattermost.com/core/channels/ux-design) and through contributions to the [Mattermost User Experience Guidelines](https://docs.mattermost.com/developer/fx-guidelines.html) on [GitHub](https://github.com/mattermost/docs/blob/master/source/developer/fx-guidelines.rst).
```
Comment (justinegeffen) [inline contributors/contributors/community.md:140]: @andrewbrown00, would you be open to reviewing this section and confirming that the links are correct?

**Review by justinegeffen — DISMISSED** [review]

File: contributors/contributors/community.md [hunk 11]
```diff
@@ -121,21 +125,21 @@ Community Coordinator: TBD
...
 
-Community Coordinator: Mike Gamble
+Community Coordinator: TBD
```
Comment (andrewbrown00) [inline contributors/contributors/community.md]: @justinegeffen you can add me as the `community coordinator` cc/ @jasonblais per our conversation.

**Review by andrewbrown00 — DISMISSED** [review]
@justinegeffen links look good and I added a comment about changing the `community contributor`

File: contributors/contributors/community.md (hunk 11 above)
Comment (justinegeffen) [inline contributors/contributors/community.md]:
```suggestion
Community Coordinator: Andrew Brown
```

**Comment by justinegeffen** [comment]
> @justinegeffen links look good and I added a comment about changing the `community contributor`

Thank you! Update made.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**